packages/rocketchat-i18n/i18n/ru.i18n.json

```diff
@@ -720,7 +720,7 @@
   "External_Service": "Внешний сервис",
   "External_Queue_Service_URL": "URL внешнего сервера очередей",
   "Facebook_Page": "Страница Facebook",
-  "False": "Ложный",
+  "False": "Нет",
```

Comment:
Interesting, so this basically means "no" in Russian and the other way was "false. :thinking:

Reply:
Original translation is correct in some cases, but in rocket chat it used for representation of Boolean value . In Russian there must be Yes/no pair.

Reply:
Okay, thanks for the clarification!